### ISSUE TITLE
Download of layers in native SRS is not correct

### ISSUE THREAD

**Issue by SterckxArnaud** [kartoza/IGRAC-GGIS#568]
I downloaded this layer (https://ggis.un-igrac.org/catalogue/#/dataset/2206) in native and WGS84 SRS. The layer in WGS84 is fine, but the one in native SRS is rotated by 90deg.
![Screenshot 2023-06-15 102255](https://github.com/kartoza/IGRAC-GGIS/assets/64771599/2ee2f807-ad69-48e2-932a-105ad415d429)

**Comment by ArnaudIGRAC** [comment]
Hi Irwan, any update on this bug?

**Comment by NyakudyaA** [comment]
I am going to check this and revert @ArnaudIGRAC

**Comment by ArnaudIGRAC** [comment]
Dear colleagues, any update on that issue?

**Comment by meomancer** [comment]
hi @NyakudyaA 
Download is directly request to geoserver
It access same URL:
https://ggis.un-igrac.org/geoserver/ows?service=WFS&outputFormat=SHAPE-ZIP&access_token=XxU2C4mtpxhGAONOG055cDiGvuZlgh

But the post is different:
**Native**:
```
<wfs:GetFeature
        service="WFS" version="1.1.0"
        xmlns:wfs="http://www.opengis.net/wfs"
        xmlns:ogc="http://www.opengis.net/ogc"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.opengis.net/wfs http://schemas.opengis.net/wfs/1.1.0/wfs.xsd">
    <wfs:Query typeName="geonode:a__2021_TBA_GGIS_utf8">
        <ogc:Filter xmlns:ogc="http://www.opengis.net/ogc">
            <ogc:And>undefined</ogc:And>
        </ogc:Filter>
    </wfs:Query>
</wfs:GetFeature>
```

**WGS84**
```
<wfs:GetFeature
        service="WFS" version="1.1.0"
        xmlns:wfs="http://www.opengis.net/wfs"
        xmlns:ogc="http://www.opengis.net/ogc"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.opengis.net/wfs http://schemas.opengis.net/wfs/1.1.0/wfs.xsd">
    <wfs:Query typeName="geonode:a__2021_TBA_GGIS_utf8" srsName="EPSG:4326">
        <ogc:Filter xmlns:ogc="http://www.opengis.net/ogc">
            <ogc:And>undefined</ogc:And>
        </ogc:Filter>
    </wfs:Query>
</wfs:GetFeature>
```

The differents are
**Native** : `<wfs:Query typeName="geonode:a__2021_TBA_GGIS_utf8">`
**WGS84**: `<wfs:Query typeName="geonode:a__2021_TBA_GGIS_utf8" srsName="EPSG:4326">`

But i checked on the geoserver, the srs is already correct:
https://ggis.un-igrac.org/geoserver/web/wicket/bookmarkable/org.geoserver.web.data.resource.ResourceConfigurationPage?4&name=a__2021_TBA_GGIS_utf8&wsName=geonode
![Selection_012](https://github.com/kartoza/IGRAC-GGIS/assets/4530905/2242c52f-74f3-4f13-8ac4-ec609a363bea)

**Comment by NyakudyaA** [comment]
@meomancer The post request is coming from GeoNode and we need to identify why it does different posts request because the two SRS native and WGS84 are using the same CRS

**Comment by NyakudyaA** [comment]
@meomancer my conclusion about this issue:
* Only shp is affected and other formats natively work well
* Native SRS is being saved well in the GeoServer configuration. 
* This doesn't happen upstream with the same layer, so we need to check how the download of shp is happening here.

Alternative solution
* Try to reproject the data from the database if it's this layer giving issues and then see what happens but if it's happening for all layers then it's a code issue from GeoNode

**Comment by ArnaudIGRAC** [comment]
We have this issue for all layers. Can you please report to GeoNode developers?

**Comment by meomancer** [comment]
hi @NyakudyaA 
I have tried using latest geonode (4.1.1), and upload new layer and it is still returning non correct one.
I think it is coming from geoserver, because the request is using ows

Current geoserver is kartoza/sadc_gip:geoserver-3.1.netcfd
Maybe we need to upgrade the geoserver? But our geoserver has netcfd, so i think we need to create latest geoserver with netcfd?

**Comment by NyakudyaA** [comment]
@meomancer You can try the following image
`kartoza/geoserver:2.23.0-geo`

Note that the env variables here should use the following coupled with other geoserver env
```
STABLE_EXTENSIONS=netcdf-out-plugin
NGINX_BASE_URL=http://geonode:80
SAMPLE_DATA=FALSE 
```
sample data = true is only helpful if you are starting from scratch and you have a volume that is empty

**Comment by meomancer** [comment]
Hi @SterckxArnaud 
This is fixed with current deployment (geoserver:2.23.0-geo-v2023.09.15)
Could you check it?
